### PR TITLE
Disable upsample optest

### PR DESCRIPTION
**Pull request by zrphercule**
Temporarily disable upsample tests.

### CODE REVIEW

**Comment by zrphercule** [comment]
@houseroad could you please give a stamp? Thanks!

File: test/onnx/test_operators.py [hunk 1]
```diff
@@ -432,9 +432,12 @@ def test_norm(self):
         x = torch.randn(1, 2, 3, 4, requires_grad=True)
         self.assertONNX(lambda x: x.norm(p=2, dim=2), (x))
 
-    def test_upsample(self):
-        x = torch.randn(1, 2, 3, 4, requires_grad=True)
-        self.assertONNX(lambda x: nn.functional.interpolate(x, scale_factor=2., mode='bilinear'), x)
+#   TODO(Rui Zhu): Enable upsample_op test by fixing the number of input.
+#   See https://github.com/onnx/onnx/pull/1467/files
+#
+#   def test_upsample(self):
```
Comment (houseroad) [inline test/onnx/test_operators.py]: Use SkipCI instead of directly commenting out the code?

**Comment by houseroad** [comment]
To unblock https://github.com/onnx/onnx/pull/1467

**Comment by zrphercule** [comment]
@houseroad done

File: test/onnx/test_operators.py [hunk 2]
```diff
@@ -55,6 +55,11 @@ def forward(self, *args):
 
 class TestOperators(TestCase):
 
+    def skipAlwaysForCI(func):
```
Comment (houseroad) [inline test/onnx/test_operators.py]: If we define this wrapper like the following, why not directly using unittest.SkipTest :-)

How about https://github.com/pytorch/pytorch/blob/cca247635c6edb323176eeac7a18d3e9ab71c558/test/onnx/test_models.py#L80

Comment (zrphercule) [reply test/onnx/test_operators.py]: great, never know such thing exist

**Review by houseroad — APPROVED** [review]
LG, if CI is green, just land it

**Comment by houseroad** [comment]
import?

**Comment by zrphercule** [comment]
@houseroad  sure.

**Pull request closed without merging**